source/catalog.html

```diff
@@ -0,0 +1,193 @@
+              <h2 class="card-item__card-title">cat energy slim 500 г</h2>
+              <table class="card-item__card-description">
+                <dt>Масса</dt>
+                <dd>500 г</dd>
+                <dt>Вкус</dt>
+                <dd>Гречка</dd>
+                <dt>Цена</dt>
+                <dd>400 ₽</dd>
+              </table>
+              <button class="card-item__button-catalog-card" type="button"><span>Заказать</span></button>
+            </a>
+          </li>
+          <li class="card-list__card-item card-item">
+            <a class="card-item__card-link" href="">
+              <img src="img/images/img-catalog-6.jpg" width="140" height="107" alt="корм с гречкой 1000 г">
+              <h2 class="card-item__card-title">cat energy slim 1000 г</h2>
+              <table class="card-item__card-description">
+                <dt>Масса</dt>
+                <dd>1000 г</dd>
+                <dt>Вкус</dt>
+                <dd>Гречка</dd>
+                <dt>Цена</dt>
+                <dd>700 ₽</dd>
+              </table>
+              <button class="card-item__button-catalog-card" type="button"><span>Заказать</span></button>
+            </a>
+          </li>
+          <li class="card-list__card-item card-item">
+            <a class="card-item__card-link" href="">
+              <img src="img/images/img-catalog-7.jpg" width="140" height="107" alt="корм с рисом 500г">
+              <h2 class="card-item__card-title">cat energy slim 500 г</h2>
+              <table class="card-item__card-description">
+                <dt>Масса</dt>
+                <dd>500 г</dd>
+                <dt>Вкус</dt>
+                <dd>Рис</dd>
+                <dt>Цена</dt>
+                <dd>500 ₽</dd>
+              </table>
+              <button class="card-item__button-catalog-card" type="button"><span>Заказать</span></button>
+            </a>
+          </li>
+        </ul>
+        <p class="catalog-products-card-header">Показать еще 100500 товаров</p>
+        <p class="catalog-products-card-text">На самом деле вкусов гораздо больше!</p>
+        <button class="catalog-product-card__button-all" type="button">Показать все</button>
+      </div>
+      <div class="catalog-products__additional-products-cards">
+        <h2 class="additional-products-title">Дополнительные товары</h2>
+        <ul class="catalog-products__additional-products-list additional-products-list">
+          <li class="additional-products-list__additional-products-item additional-products-item">
+            <a class="additional-products-item__additional-products-link" href="">
+            <h3 class="additional-products-item__title">сахарозаменитель</h3>
+            <p class="additional-products-item__text">1 упаковка (100 г)</p>
+            <p class="additional-products-item__price">200 ₽</p>
+            <button class="additional-products-item__additional-button" type="button"><span>Заказать</span></button>
+            </a>
+          </li>
```

Comment:
У всех карточек так.

```suggestion
          <li class="additional-products-list__additional-products-item additional-products-item">
            <h3 class="additional-products-item__title">сахарозаменитель</h3>
            <p class="additional-products-item__text">1 упаковка (100 г)</p>
            <p class="additional-products-item__price">200 ₽</p>
            <a class="additional-products-item__additional-button" type="#">Заказать</a>
          </li>
```